packages/tools/client-debugger/client-debugger/src/FluidDebuggerLogger.ts

```diff
@@ -81,13 +140,16 @@ export class FluidDebuggerLogger extends TelemetryLogger {
 		// TODO: ability to disable the logger so this becomes a no-op
 
 		const newEvent: ITelemetryBaseEvent = this.prepareEvent(event);
-
-		postMessagesToWindow<TelemetryEventMessage>(undefined, {
+		const newMessage: TelemetryEventMessage = {
 			source: debuggerMessageSource,
 			type: "TELEMETRY_EVENT",
 			data: {
-				contents: newEvent,
+				contents: [newEvent],
 			},
-		});
+		};
+
+		// insert log into the beginning of the array to show the latest log first
+		this._telemetryLog.unshift(newEvent);
+		postMessagesToWindow<TelemetryEventMessage>(undefined, newMessage);
```

Comment:
Do we have logging options we can pass here (first argument)?

Reply:
We do but I deliberately left them undefined and asked @chentong7 to keep it that way; otherwise we spam the console when there's a burst of telemetry events, which ends up happening almost every time there's any change to fluid data. For other messages I think it's fine to log them by default for now; these were too noisy. Once everything is working as expected, I'd like to see all that logging go away though, or at least be configurable.

Reply:
Ah, okay. Can we leave a brief comment explaining why we aren't passing it then? That way no one accidentally re-adds it in the future :)